test/e2e/package.json

```diff
@@ -1,15 +1,17 @@
 {
   "name": "test-e2e",
   "version": "1.0.0",
-  "description": "End-to-end tests for the Django REST Boilerplate application",
+  "description": "End-to-end tests for the EID project",
```

Comment:
Please decide for a name and update accordingly. Plain "EID" is misleading, as this repository shall provide an EID based authentication service.